usbtool.c

```diff
@@ -27,68 +27,87 @@
 
 enum { INFO, WARNING, ERROR };
 
-void logmsg(int s, char *msg, ...) {
+int isopt(int argc, char **argv, const char *opt){
+
+    for(int i = 1; i < argc; ++i){
```

Comment:
you can probably omit { } in for to make code shorter

Reply:
Yes! :D